client/src/app/submission/submission.component.html

```diff
@@ -124,6 +152,24 @@ <h2 class="vf-text vf-text-heading--2">
         </app-files>
       </ng-template>
     </mat-tab>
+
+    <mat-tab label="Spreadsheet">
+      <ng-template matTabContent>
+        <div class="vf-u-margin__top--xxl">
+          <h3 class="vf-text vf-text-heading--4">Import a spreadsheet</h3>
+          <p>You could import a spreadsheet to update the metadata in this submission.
+            The spreadsheet should contain the uuid's of the metadata you want to update.</p>
```

Comment:
What does _should_ mean here? Will the spreadsheet contain uuids or will it not?
```suggestion
            The spreadsheet will contain the uuid's of the metadata you want to update.</p>
```

Reply:
I think @aaclan-ebi meant that the metadata's UUIDs should be there as without them impossible to know what to update. I think the `will` is not good in that sentence.

Reply:
The spreadsheet could be either manually created or could be downloaded. It's just a note that whatever spreadsheet that the user will use should contain UUIDs of the metadata they're updating.

Reply:
Yes, I got that. I just meant this sentence is good as it is.